subjects/java/projects/sharpen-it/README.md

```diff
@@ -0,0 +1,61 @@
+## Sharpen-It
+
+### Objectives
+
+In this project, your task is to finalize the provided environment for your project, applying coding best practices to enhance clarity and cleanliness.
+
+### Instructions
+
+You have to download this [Environment](./env.zip), then you will find two folders, one for Jenkins and another for SonarQube. Your goals finish up all the messing configurations and make the environment functional
+
+#### Jenkins
+
+- Build and run the Jenkins Docker image.
+
+- Install missing plugins.
+
+- Set up credentials to access your repository.
+
+- Fix and test the build and testing processes for both the back-end and front-end.
```

Comment:
This sentence is not clear to me. Is the below sentence equivalent to the required task?
```
Fix the provided pipeline to build and test the back-end and front-end
```

Reply:
this is a duplicate comment.